.circleci/config.yml

```diff
@@ -23,9 +23,8 @@ jobs:
             FULL_SKIP1=$SKIP_FILES1$SKIP_FILES2$SKIP_FILES3$SKIP_FILES4
             FULL_SKIP2=$SKIP_FILES5$SKIP_FILES6$SKIP_FILES7$SKIP_FILES8
             SKIP_FILES=$FULL_SKIP1$FULL_SKIP2
-            #source ./.ci/common.sh
-            #should_run_job "recheck" $SKIP_FILES
-            export RUN_JOB=1
+            source ./.ci/common.sh
+            should_run_job "recheck" $SKIP_FILES
```

Comment:
lets have code like:
```
export RUN_JOB=1
source ./.ci/common.sh
# should_run_job will change RUN_JOB variable
should_run_job "recheck" $SKIP_FILES
```

attention on `export RUN_JOB=1` it is good to stay that by default we plan execution, if any problems/error happens at `should_run_job` we will do execution. So we will not block any PRs and we will just have more execution then actually not required, so if we notice a problem (performance issue of CI queue) we can investigate it.

Reply:
If an error happens in should_run_job, it will kill whole build as it is an error and goes all the way up the chain.

Reply:
no that much guaranty, wrapper shell need to have something like "-x", but we do not control how our commands are executed.

Reply:
I applied this change to the code., just to be a bit more sure.

ones CI pass, I will merge and start testing by test PRs.